### PR TITLE
Implemented Sanity CMS & More Pages

### PR DESCRIPTION
**Pull request by NickBurnett**
- added join, events, programs, sponsors, and scholarship pages
- added Sanity CMS for all content injections (few hardcoded sections, will change before prod)
- lost the code formatting battle to a formidable opponent...

### CODE REVIEW

File: components/DevProject.js [hunk 1]
```diff
@@ -4,15 +4,101 @@ import styles from '../styles/component/DevProject.module.css';
...
+        <div key={d} className={styles.overlay_about_paragraph}>
+          {d}
+        </div>
+      )
+    );
+    return (
+      <div className={styles.overlay}>
+        <div className={styles.overlay_card}>
+          <div className={styles.overlay_exit}>
+            <svg
```
Comment (WillieCubed) [inline components/DevProject.js:26]: Instead of directly embedding SVG into the page, consider making a component.

File: components/ProgramPage.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
+import { Component } from 'react';
+import Info from '../components/Info';
+import ImageCarousel from './ImageCarousel';
+import styles from '../styles/component/ProgramPage.module.css';
+export class Testimonial extends Component {
```
Comment (WillieCubed) [inline components/ProgramPage.js:12]: What does this component do?

File: components/ProgramPage.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          <div className={styles.testimonial_description}>
+            {this.props.description}
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+export default class ProgramPage extends Component {
```
Comment (WillieCubed) [inline components/ProgramPage.js:62]: What does this component do? Describe the props.

File: components/TeamPage.js [hunk 4]
```diff
@@ -5,37 +5,74 @@ import Timeline from './Timeline';
 import styles from '../styles/component/TeamPage.module.css';
 
 export class Profile extends Component {
```
Comment (WillieCubed) [inline components/TeamPage.js:16]: What does this component do? Describe the props.

**Comment by WillieCubed** [comment]
#### 📝Changed routes:


| [**/events**](https://acmutd-website-kl01a3mow-acmutd.vercel.app/events) | [**/**](https://acmutd-website-kl01a3mow-acmutd.vercel.app/) |
|:-:|:-:|
|<a href="https://acmutd-website-kl01a3mow-acmutd.vercel.app/events">Screenshot of /events</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/DohQqBbVPjingeptPMNkTU4l30w2PZTDygkrPbeod26HFUPn1F0Sx6oRHvVsaZtFSXl0J3P3NUOKNaCDuN0lNw.png">(view full size)</a> |<a href="https://acmutd-website-kl01a3mow-acmutd.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/zTam3a5pCN2XiDBEuM-zMg_oVqWy1wlTxBgj9NvnjOzaqZXPx0LwREOlZ_gRUfuHv81zsAVTrLJm8zBmoM3YTA.png">(view full size)</a> |



| [**/join**](https://acmutd-website-kl01a3mow-acmutd.vercel.app/join) | [**/program/projects**](https://acmutd-website-kl01a3mow-acmutd.vercel.app/program/projects) |
|:-:|:-:|
|<a href="https://acmutd-website-kl01a3mow-acmutd.vercel.app/join">Screenshot of /join</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/Hl10Vforlafu-wOqXJyauR-HdWXqiGELjHe_7t-cKDgbMkuSEQ0DWdmnMJDaI8BbeqdPa5AlD5LKzSr4KLz8cA.png">(view full size)</a> |<a href="https://acmutd-website-kl01a3mow-acmutd.vercel.app/program/projects">Screenshot of /program/projects</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/FW3yoYjLE5Y1feRCyJJrFe9UQxtWYIZmH6P_HSgQ88UNJ-ruzehMWD0vFTlFMrVNFNhAKfvF7vgb9xtrII85lA.png">(view full size)</a> |



| [**/program/tip**](https://acmutd-website-kl01a3mow-acmutd.vercel.app/program/tip) | [**/scholarship**](https://acmutd-website-kl01a3mow-acmutd.vercel.app/scholarship) |
|:-:|:-:|
|<a href="https://acmutd-website-kl01a3mow-acmutd.vercel.app/program/tip">Screenshot of /program/tip</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/VSAP852sgU-T2Om57CmiShWtONKgQgiJBjFsvmyAmHG16az5vhnsj-dLymcnmXXIJmaQJOKIT0rzDaEsYn6Luw.png">(view full size)</a> |<a href="https://acmutd-website-kl01a3mow-acmutd.vercel.app/scholarship">Screenshot of /scholarship</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/mY6Ir2drnHfNUqT4Cm4fMcrMPKRBUo8HeLQiN_Xb5UfkdHo40Jjs5RWLml1tchodnV62rsEgkqFDVQml5Vo9Fg.png">(view full size)</a> |



And 5 other routes:
- [**/sponsors**](https://acmutd-website-kl01a3mow-acmutd.vercel.app/sponsors)
- [**/team/dev**](https://acmutd-website-kl01a3mow-acmutd.vercel.app/team/dev)
- [**/team/education**](https://acmutd-website-kl01a3mow-acmutd.vercel.app/team/education)
- [**/team/industry**](https://acmutd-website-kl01a3mow-acmutd.vercel.app/team/industry)
- [**/team/media**](https://acmutd-website-kl01a3mow-acmutd.vercel.app/team/media)

1 deleted route:
- **/dev**

Commit 9071e8883e4108e830cd12fb950132d4509d9e30 (https://acmutd-website-kl01a3mow-acmutd.vercel.app).

**Comment by NickBurnett** [comment]
> #### 📝Changed routes:
> [**/events**](https://acmutd-website-2oui55a3b-acmutd.vercel.app/events)	[**/**](https://acmutd-website-2oui55a3b-acmutd.vercel.app/)
> [Screenshot of /events](https://acmutd-website-2oui55a3b-acmutd.vercel.app/events)
> [(view full size)](https://images.weserv.nl?url=https://sjc.microlink.io/yit7SGUTEP-amzW7uSNHukJ0dE9T8Fo9c3_eL6dOjRpGjRgAYcybaQiR6p2jAk-pQxTFGn_9TngPLXZoeRiyAg.png)	[Screenshot of /](https://acmutd-website-2oui55a3b-acmutd.vercel.app/)
> [(view full size)](https://images.weserv.nl?url=https://sjc.microlink.io/f1_JLj_gnMY10yttWk1Yu1IxL0ro7sjK_qTLKkyCpncIU9TadxCcbJkK4Fpy6jmA7JvGVYOcLI8NVvOxLgJUvw.png)
> [**/join**](https://acmutd-website-2oui55a3b-acmutd.vercel.app/join)	[**/program/projects**](https://acmutd-website-2oui55a3b-acmutd.vercel.app/program/projects)
> [Screenshot of /join](https://acmutd-website-2oui55a3b-acmutd.vercel.app/join)
> [(view full size)](https://images.weserv.nl?url=https://sjc.microlink.io/pvqCGdq-roF54TISVvQVMd9BfswjllHjKsRr9jyG8_9Mg0jt_S8HSj5myq6rD65ryjav6YUIF7rrZDNkxJmLjg.png)	[Screenshot of /program/projects](https://acmutd-website-2oui55a3b-acmutd.vercel.app/program/projects)
> [(view full size)](https://images.weserv.nl?url=https://sjc.microlink.io/fCDdRvgMTI-_KHL_HBUB-m_KQmN62TFCxs9p0irTEbRl6XAH_XeWQHpQLCoKiwbv71_8A46rHJFXrkWa_ZCpkQ.png)
> [**/program/tip**](https://acmutd-website-2oui55a3b-acmutd.vercel.app/program/tip)	[**/scholarship**](https://acmutd-website-2oui55a3b-acmutd.vercel.app/scholarship)
> [Screenshot of /program/tip](https://acmutd-website-2oui55a3b-acmutd.vercel.app/program/tip)
> [(view full size)](https://images.weserv.nl?url=https://sjc.microlink.io/uqPjdwFkvZ-xGqroRi9QFG4rVmSX-UdaaHc1KqMf5LE5kRSlaRJ1togXuYdsidezZxgBT1zdTTNDL2SSKKUtjA.png)	[Screenshot of /scholarship](https://acmutd-website-2oui55a3b-acmutd.vercel.app/scholarship)
> [(view full size)](https://images.weserv.nl?url=https://sjc.microlink.io/5jpebq48IWKY8higfoq70ELtpu4OAXD4_KPABF4UXsivNNRhNvnime_GbvH6zVE-iClhBg8YWKNOkjF5IJrFJg.png)
> And 5 other routes:
> 
> * [**/sponsors**](https://acmutd-website-2oui55a3b-acmutd.vercel.app/sponsors)
> * [**/team/dev**](https://acmutd-website-2oui55a3b-acmutd.vercel.app/team/dev)
> * [**/team/education**](https://acmutd-website-2oui55a3b-acmutd.vercel.app/team/education)
> * [**/team/industry**](https://acmutd-website-2oui55a3b-acmutd.vercel.app/team/industry)
> * [**/team/media**](https://acmutd-website-2oui55a3b-acmutd.vercel.app/team/media)
> 
> 1 deleted route:
> 
> * **/dev**
> 
> Commit [75b4731](https://github.com/acmutd/acmutd.github.io/commit/75b4731efcc47d581db1eb446210d16105e1054d) (https://acmutd-website-2oui55a3b-acmutd.vercel.app).

What does this do?

File: .prettierrc.js [hunk 5]
```diff
@@ -3,6 +3,6 @@ module.exports = {
   semi: true,
   singleQuote: true,
   tabWidth: 2,
-  trailingComma: 'all',
```
Comment (WillieCubed) [inline .prettierrc.js]: Trailing commas are actually better for version control purposes, so I'd revert this change.

**Review by WillieCubed — CHANGES_REQUESTED** [review]

File: components/DevProject.js (hunk 1 above)
Comment (NickBurnett) [reply components/DevProject.js:26]: This will happen later tonight once I have a good system for SVGs. Resolving so that this can merge faster.
